src/HonzaBotner.Discord.Services/Commands/ReminderCommands.cs

```diff
@@ -97,15 +102,9 @@ private DiscordEmbed CreateReminderEmbed(CommandContext context, Reminder remind
             return new DiscordEmbedBuilder()
                 .WithAuthor(context.Member.RatherNicknameThanUsername(), null, context.Member.AvatarUrl)
                 .WithColor(DiscordColor.Blurple)
-                .WithTitle("Reminder created.")
-                .WithDescription(
-@$"If you would like to cancel this reminder, click the {_options.CancelEmojiName} reaction.
```

Comment:
Also, why are the description, the content, and the owner removed here?

Reply:
It's not bad itself, but from the perspective of BFU, or even advanced, I would only need some text, not multiple fields. Aka one string is enough and provides a simpler API to be remembered.

Reply:
Description of how to join is removed because it seemed that it just uses too much space and people will know how it works after some usages, the same as reacting with 📌. 

The Author field is already mentioned in the footer of the embed, therefore it was a useless mention?

Reply:
imo the content can be useful and as it is an optional parameter doesn't have to be used by BFU, 
\> BFU
\> FIT
choose one

Reply:
Without the content, there is nothing to provide info about why the reminder was created in the first place

Reply:
I don't understand. If you wanna create a reminder, you need 2 essential things. The first is obviously the datetime, the second one is some text/reason.  Separating the text into multiple parts is overengineering since you can fit everything in one text field. 

If you intended the fields differently, please elaborate. 😊🙏

Reply:
Uh oh, my bad, forgot about the `title` field

Reply:
Then I think the `content` field can be dropped from the database as well (this will probably require a new EF migration)